### PR TITLE
Clean up icon

### PR DESCRIPTION
**Pull request by koddsson**

### CODE REVIEW

File: packages/cf-component-icon/README.md [hunk 1]
```diff
@@ -12,20 +12,66 @@ $ npm install cf-component-icon
 
 ```jsx
 import React from 'react';
-import Icon, { TYPES, SIZES } from 'cf-component-icon';
```
Comment (prestonp) [inline packages/cf-component-icon/README.md]: Keep `import Icon from 'cf-component-icon';` ?

Comment (koddsson) [inline packages/cf-component-icon/README.md]: Ha! Good catch!

File: packages/cf-component-icon/example/basic/component.js [hunk 2]
```diff
@@ -1,18 +1,64 @@
 import React from 'react';
-import Icon, { TYPES, SIZES } from 'cf-component-icon';
```
Comment (prestonp) [inline packages/cf-component-icon/example/basic/component.js]: Here too!

File: packages/cf-component-icon/src/Icon.js [hunk 3]
```diff
@@ -25,10 +24,57 @@ class Icon extends React.Component {
...
+    'facebook',
+    'file',
+    'filter',
+    'flowchart',
+    'gear',
+    'googleplus',
+    'hamburger',
+    'happy',
+    'help',
+    'info-sign pause time',
```
Comment (prestonp) [inline packages/cf-component-icon/src/Icon.js]: Separate these types

File: packages/cf-component-icon/README.md [hunk 4]
```diff
@@ -12,20 +12,66 @@ $ npm install cf-component-icon
...
 
 const IconComponent = () => (
   <div>
     <p>Types:</p>
     <p>
-      {TYPES.map(type => <Icon label={type} type={type} key={type} />)}
+      <Icon label="bolt" type="bolt" key="bolt" />
+      <Icon label="calendar" type="calendar" key="calendar" />
+      <Icon label="caret-down" type="caret-down" key="caret-down" />
+      <Icon label="caret-left" type="caret-left" key="caret-left" />
```
Comment (koddsson) [inline packages/cf-component-icon/README.md]: All these keys aren't needed anymore

**Pull request merged**